lib/winston/common.js

```diff
@@ -85,7 +85,7 @@ exports.clone = function (obj) {
       copy[key] = obj[key];
     });
 
-    return copy;
+    return cycle.decycle(copy);
```

Comment:
since we're decycling in the base case on line 97 anyway, this is probably fine and wouldn't degrade performance

Reply:
The base case would never be invoked because this `return` is a short circuit.